app/waf/src/controllers/adc.controller.ts

```diff
@@ -15,9 +15,11 @@ import {
   put,
   del,
   requestBody,
+  HttpErrors,
 } from '@loopback/rest';
 import {Adc} from '../models';
 import {AdcRepository} from '../repositories';
+import uuid = require('uuid');
```

Comment:
is 'uuid' a javascript package rather than typescript package? why do we choose this package.

Reply:
it is generated by tslint, uuid should an internal npm package, better to use it..